package.json

```diff
@@ -22,7 +22,7 @@
     "build:test": "npm-run-all build:test:browserify build:test:webpack",
     "clean": "npm-run-all -p clean:*",
     "clean:build": "rm -rf dist dist-test es5 && mkdir -p dist dist-test es5",
-    "lint": "jshint src/**/*.js",
+    "lint": "jshint src/**/*.js test/*.js",
```

Comment:
Maybe `test/**/*.js` to account for subdirs.

Reply:
That was actually what I did at first, and was surprised when the linting failed:
```
> jshint src/**/*.js test/**/*.js

ERROR: Can't open test/**/*.js
```
It's not using `vjsstandard`, so maybe the underlying linter is different?

Reply:
Tested locally, it seems that at least this version of jshint breaks if it can't find an subdirectories. Since we don't have any right now, and we may change to use vjsstandard in the future, I think this is fine for now.